src/lib/server/endpoints/tgi/endpointTgi.ts

```diff
@@ -11,14 +16,32 @@ export const endpointTgiParametersSchema = z.object({
 	url: z.string().url(),
 	accessToken: z.string().default(env.HF_TOKEN ?? env.HF_ACCESS_TOKEN),
 	authorization: z.string().optional(),
+	multimodal: z
+		.object({
+			// Assumes IDEFICS
+			image: createImageProcessorOptionsValidator({
+				supportedMimeTypes: ["image/jpeg", "image/webp"],
+				preferredMimeType: "image/webp",
+				maxSizeInMB: 5,
+				maxWidth: 224,
+				maxHeight: 224,
+			}),
+		})
+		.default({}),
```

Comment:
why is `default` being used instead of `optional`?

Reply:
for each provider, we set sensible defaults so that the user likely doesn't need to configure it. you can check out the `createImageProcessorOptionsValidator` function for more on that